scripts/bravo_csv.py

```diff
@@ -242,6 +259,29 @@ def prepooling(currentStep, lims):
         logging.info("Work done")
 
 
+def setup_qpcr(currentStep, lims):
+    log = []
+    data = aliquot_fixed_volume(currentStep, lims, MIN_WARNING_VOLUME, log)
+    with open("bravo.csv", "w") as csvContext:
+        for s in data:
+            csvContext.write("{0},{1},{2},{3},{4}\n".format(s['src_fc'], s['src_well'], s['vol'], s['dst_fc'], s['dst_well']))
+    if log:
+        with open("bravo.log", "w") as logContext:
+            logContext.write("\n".join(log))
+    for out in currentStep.all_outputs():
+        # attach the csv file and the log file
+        if out.name == "EPP Generated Bravo CSV File":
+            attach_file(os.path.join(os.getcwd(), "bravo.csv"), out)
+        if log and out.name == "Bravo Log":
+            attach_file(os.path.join(os.getcwd(), "bravo.log"), out)
+    if log:
```

Comment:
This will never trigger as nothing is ever written to the log in `aliquot_fixed_volume`